src/utils/event-utils.ts

```diff
@@ -256,9 +256,7 @@ export const _info = {
         if (!document?.referrer) {
             return '$direct'
         }
-        const parser = document.createElement('a') // Unfortunately we cannot use new URL due to IE11
-        parser.href = document.referrer
-        return parser.host
+        return convertToURL(document.referrer)?.host || '$direct'
```

Comment:
@robbie-c I moved this method into utils as I was about to write the same thing. Want to review? I'm not sure I chose the right fallback value here either